<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>1098</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Документ</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>1098</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/ПокупкаJPG.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border-radius: 25px;
font: 15pt &quot;Helvetica&quot;;
color: rgb(252, 0, 6);
</string>
   </property>
   <property name="text">
    <string>Х закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="operation">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>16-12-2021</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>16-12-2021</string>
   </property>
  </widget>
  <widget class="QLabel" name="curr_code">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="curr_sum">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="rub_sum">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="rate">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_num">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_num_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="curr_sum_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="rub_sum_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>710</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="rate_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>740</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="doc_buy.qrc"/>
 </resources>
 <connections/>
</ui>
